src/main/java/de/tum/in/www1/artemis/domain/CourseCodeOfConductAgreement.java

```diff
@@ -0,0 +1,56 @@
+package de.tum.in.www1.artemis.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonIncludeProperties;
+
+/**
+ * A user's agreement of a course's code of conduct.
+ */
+@Entity
+@Table(name = "course_code_of_conduct_agreement")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class CourseCodeOfConductAgreement extends DomainObject {
+
+    @ManyToOne
+    @JsonIncludeProperties({ "id" })
+    @NotNull
+    private Course course;
+
+    @ManyToOne
+    @JsonIncludeProperties({ "id" })
+    @NotNull
+    private User user;
+
+    @Column(name = "is_accepted")
+    private Boolean isAccepted;
```

Comment:
Can you make this a primitive boolean and adjust the getter and setter correspondingly? I think you can also initialize the field with false.

Reply:
Agree 59b450f4a596ec5d8e9361b399de2d32a1d5ec48